src/pages/EnablePayments/IdologyQuestions.tsx

```diff
@@ -126,13 +106,13 @@ function IdologyQuestions({questions, idNumber}) {
                 <Text style={styles.mb3}>{translate('additionalDetailsStep.helpTextIdologyQuestions')}</Text>
                 <TextLink
                     style={styles.mb3}
-                    href="https://use.expensify.com/usa-patriot-act"
+                    href={CONST.HELP_LINK_URL}
                 >
                     {translate('additionalDetailsStep.helpLink')}
                 </TextLink>
             </View>
             <FormProvider
-                formID={ONYXKEYS.WALLET_ADDITIONAL_DETAILS}
+                formID={ONYXKEYS.FORMS.IDOLOGY_QUESTIONS_FORM}
```

Comment:
We had used the `ONYXKEYS.WALLET_ADDITIONAL_DETAILS` key for IdologyQuestions Form,
https://github.com/Expensify/App/blob/f50ac2af5194c9b3c8ba538180742972c8a41700/src/pages/EnablePayments/IdologyQuestions.js#L134-L135

But in this PR I have created a new key `ONYXKEYS.FORMS.IDOLOGY_QUESTIONS_FORM` for the  IdologyQuestions, but now I thinking that server erorr will not be visible for this Form and I think we have to use the previous(WALLET_ADDITIONAL_DETAILS) key here, @ntdiary you have any other thought on this change ?

cc: @blazejkustra

Reply:
I think you are right and we have to use the same key. Let's add `ONYXKEYS. FORMS.WALLET_ADDITIONAL_DETAILS` with the same value for the key as `ONYXKEYS.WALLET_ADDITIONAL_DETAILS` has.

You will have to use  `// @ts-expect-error` in some places but that's fine as there will be conflict in these types:`ONYXKEYS.WALLET_ADDITIONAL_DETAILS` vs `ONYXKEYS. FORMS.WALLET_ADDITIONAL_DETAILS`

Reply:
@blazejkustra Removed IdologyQuestionsForm and now we are using the current AdditionalDetailStepForm and We do not have to add // @ts-expect-error anywhere, as `IdologyQuestions` page does not use the any Onyx key, @ntdiary we have added above the mentioned changes, PR is ready for the review